### PR TITLE
added luac maker for lua

### PR DESCRIPTION
**Pull request by sbdchd**

### CODE REVIEW

File: autoload/neomake/makers/ft/lua.vim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function! neomake#makers#ft#lua#EnabledMakers()
-    return ['luacheck']
+    return ['luacheck', 'luac']
```
Comment (blueyed) [inline autoload/neomake/makers/ft/lua.vim]: Should this come first, because it's faster and does basic syntax checks only?

**Comment by blueyed** [comment]
I don't think both should be enabled by default, since `luacheck` seems to report the same fatal parse errors, doesn't it?

So I suggest to use only `luacheck` by default if it exists.
Do you agree?

**Comment by sbdchd** [comment]
@blueyed sounds like a plan

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**